src/sentry/integrations/jira/integration.py

```diff
@@ -541,10 +541,23 @@ def fetch_issue_create_meta(self, client, project_id):
             )
         return meta
 
-    def get_create_issue_config_no_params(self):
-        return self.get_create_issue_config(None, None, params={})
-
     def get_create_issue_config(self, group, user, **kwargs):
+        """
+        We use the `group` to get three things: organization_slug, project
+        defaults, and default title and description. In the case where we're
+        getting `createIssueConfig` from Jira for Ticket Rules, we don't know
+        the issue group beforehand.
+
+        :param group: (Optional) Group model.
+        :param user: User model. TODO Make this the first parameter.
+        :param kwargs: (Optional) Object
+            * params: (Optional) Object
+            * params.project: (Optional) String. The Jira project ID.
```

Comment:
@mgaeta I think this doc string is wrong because it's the Sentry project, yea?

Reply:
It reads `"project"` as an ID here:
```
project_id = params.get("project", defaults.get("project"))
```
and it think it is also saved as an ID here: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/issues.py#L124

Reply:
@mgaeta and that's the Sentry project, correct? We are using `group.project` when calling `store_issue_last_defaults`.

Reply:
You're right let me update the docstring.